04-spatial-operations.Rmd

```diff
@@ -41,53 +46,52 @@ It is important to note that spatial operations that use two spatial objects rel
 ## Spatial subsetting
 
 Spatial subsetting is the process of selecting only those features of a spatial object that in some way *intersect* with another spatial object.
+<!-- can we already reference the intersection figure here? Generally asking, why can't we present the other topological operations here? Would fit nicely in this section... -->
 Note that 'intersect' in this context has a precise meaning:
 if `y` is used to subset features in a 'target' object of `x`, any features in `x` that touch, overlap or are within features in `y` will be selected.
 Intersect is the default operation for spatial subsetting but others can be used using the `op =`
 argument.^[Interested
 readers can see this default value of `op` set in the first line of the function call by entering its long-form name into the console: 
-`` sf:::`[.sf` ``
+`` sf:::`[.sf` `` 
+The `?sf` help page documents this also.
```

Comment:
Does this footnote work OK? if so, great addition but I think there should be a period before the capital T in The.

Reply:
changed that